image_proc/include/rectify.hpp

```diff
@@ -31,47 +31,43 @@
 *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 *  POSSIBILITY OF SUCH DAMAGE.
 *********************************************************************/
-#ifndef IMAGE_PROC_PROCESSOR_H
-#define IMAGE_PROC_PROCESSOR_H
+#include <thread>
```

Comment:
Good work on adding includes for the std:: objects that you use. However, you seem to have lost the header-guard in the process. Please re-add.